README.md

```diff
@@ -342,8 +342,8 @@ Integrated to Azure DevOps release pipeline for the ci_prod branch. With this, f
 
 When releasing the agent, we have a separate Azure DevOps pipeline which needs to be run to publish the image to prod MCR and our PROD AKS clusters. 
 
-For development, agent image will be in this format mcr.microsoft.com/azuremonitor/containerinsights/cidev:`<MM><DD><YYYY>`-<git-commit-id>.
-For releases, agent will be in this format mcr.microsoft.com/azuremonitor/containerinsights/ciprod:ciprod`<MM><DD><YYYY>-<git-commit-id>`.
+For development, agent image will be in this format mcr.microsoft.com/azuremonitor/containerinsights/cidev:`Major.Minor.Patch`-CommitAheadCount-<git-commit-id>.
+For releases, agent will be in this format mcr.microsoft.com/azuremonitor/containerinsights/ciprod:`Major.Minor.Patch`.
```

Comment:
lets cover formats for both windows & Linux images.

Reply:
added